docs/docs/20-usage/45-cron.md

```diff
@@ -0,0 +1,29 @@
+# Cron
+
+To enable cron jobs you need at least have push access to the repo.
+
+:::warning
+by default cron wont trigger any step in pipelines, as the default event branch filters it.  
+Read more at: [pipeline-syntax#event](/docs/usage/pipeline-syntax#event)
+:::
+
+1. So to start add the event filter to all steps where you like to run:
+
+    ```diff
+     pipeline:
+       sync_locales:
+         image: weblate_sync
+         settings:
+           url: example.com
+           token:
+             from_secret: weblate_token
+    +    when:
+    +      event: cron
+    ```
+
+2. Create a new cron job at repo settings
+
+    ![cron settings](./cron-settings.png)
+
+    Schedule syntax can be found at https://pkg.go.dev/github.com/robfig/cron?utm_source=godoc#hdr-CRON_Expression_Format.  
+    Examples: `@every 5m`, `@daily`, `0 30 * * * *` ...
```

Comment:
```suggestion
    The supported schedule syntax can be found at <https://pkg.go.dev/github.com/robfig/cron?utm_source=godoc#hdr-CRON_Expression_Format>. 
    
    Examples: `@every 5m`, `@daily`, `0 30 * * * *` ...

    :::info
    Woodpeckers cron syntax starts with seconds instead of minutes (used by most linux cron-jobs).  Example: "At minute 30 every hour" would be `0 30 * * * *` instead of `30 * * * *` 
    :::
```